widget/playground/src/components/SideNavigation/SideNavigation.tsx

```diff
@@ -13,7 +14,6 @@ const SIDE_NAVIGATION_TABS = [
     title: 'Functional Setting',
     id: SIDE_TABS_IDS.FUNCTIONAL,
     icon: <SettingsIcon color="secondary" />,
```

Comment:
The size of sidebar icons doesn't compatible with Figma. (The size of the icons is 24 in Figma)

Reply:
fixed.